fluent-react/src/localization.ts

```diff
@@ -73,6 +79,144 @@ export class ReactLocalization {
     return fallback || id;
   }
 
+  getElement(
+    componentToRender: ReactElement,
```

Comment:
As the type indicates, this argument is an "element". Calling it such would be clearer, maybe something like `sourceElement`?

Reply:
Yeah that works. Can get lost in the weeds when moving code around from one place to another :sweat_smile: dbfbdb53